src/lightning/data/__init__.py

```diff
@@ -1,3 +1,4 @@
+from lightning.data.constants import RequirementCache
```

Comment:
Shouldn't this be imported from lightning_utilities?

Reply:
done !